EXE/flhook_plugins/FLSR-LootBoxes.cfg

```diff
@@ -22,3 +22,111 @@ fail_sound_nickname = ui_close_infocard
+
+[General]
+success_sound_nickname = ui_gain_level
+fail_sound_nickname = ui_close_infocard
+
+
+;; ---------------------
+;; DROPABLE BOXES
+;; ---------------------
+
+[LootBox]
+box_nickname = surprise_box_loot_bronze01
+loot_item = blueprint_cv_gun_laser_light02, 25
+loot_item = blueprint_cv_gun_photon_light02, 25
+loot_item = blueprint_gd_im_gun_tachyon_light02, 25
+loot_item = commodity_lots_of_credits, 25
+
+[LootBox]
+box_nickname = surprise_box_loot_silver01
+loot_item = blueprint_cv_gun_laser_medium02, 20
+loot_item = blueprint_cv_gun_photon_medium02, 20
+loot_item = blueprint_gd_im_gun_tachyon_medium02, 20
+loot_item = blueprint_cv_gun_particle_heavy02, 20
+loot_item = blueprint_cv_gun_plasma_heavy02, 20
+
+[LootBox]
+box_nickname = surprise_box_loot_gold01
+loot_item = blueprint_cv_power_light_opt01, 15
+loot_item = blueprint_cv_power_medium_opt01, 15
+loot_item = blueprint_cv_power_heavy_opt01, 15
+loot_item = blueprint_ge_cloak_fighter_std01, 2
+loot_item = material_gunboat_data_recorder, 30
+loot_item = material_cruiser_data_recorder, 15
+loot_item = material_battleship_data_recorder, 8
+
+;; ---------------------
+;; REWARD BOXES
+;; ---------------------
+
+[LootBox]
+box_nickname = surprise_box_blueprint_equipment01
+loot_item = blueprint_cv_power_light_opt01, 10
+loot_item = blueprint_cv_power_medium_opt01, 10
+loot_item = blueprint_cv_power_heavy_opt01, 10
+loot_item = blueprint_ge_cloak_capital_light_std01, 10
+loot_item = blueprint_ge_cloak_capital_medium_std01, 10
+loot_item = blueprint_ge_cloak_capital_heavy_std01, 10
+loot_item = blueprint_ou_engine_transport01, 10
+loot_item = blueprint_ou_engine_train01, 10
+loot_item = blueprint_corsair_engine_transport01, 10
+loot_item = blueprint_corsair_engine_train01, 10
+
+[LootBox]
+box_nickname = surprise_box_blueprint_guns01
+loot_item = blueprint_special_gun01, 3
+loot_item = blueprint_special_gun02, 3
+loot_item = blueprint_special_gun03, 3
+loot_item = blueprint_special_gun04, 3
+loot_item = blueprint_special_gun05, 3
+loot_item = blueprint_special_gun06, 3
+loot_item = blueprint_special_gun07, 3
+loot_item = blueprint_special_gun08, 3
+loot_item = blueprint_special_gun09, 3
+loot_item = blueprint_special_gun10, 3
+loot_item = blueprint_special_gun11, 3
+loot_item = blueprint_special_gun12, 3
+loot_item = blueprint_special_gun13, 3
+loot_item = blueprint_special_gun14, 3
+loot_item = blueprint_special_gun15, 3
+loot_item = blueprint_special_gun16, 3
+loot_item = blueprint_special_gun17, 3
+loot_item = blueprint_special_gun18, 3
+loot_item = blueprint_special_gun19, 3
+loot_item = blueprint_special_gun20, 3
+loot_item = blueprint_special_gun21, 3
+loot_item = blueprint_sp_particle_light01, 3
+loot_item = blueprint_sp_laser_heavy01, 3
+loot_item = blueprint_gun_codename_plasma_special01, 3
+loot_item = blueprint_gun_codename_plasma_special02, 3
+loot_item = blueprint_gun_codename_plasma_special03, 3
+loot_item = blueprint_gun_codename_repair_special01, 3
```

Comment:
Why is there 4 times the same blueprint in a row?

Reply:
those strings say 01, 02, 03 at the end, they are different

Reply:
```
loot_item = blueprint_gun_codename_repair_special01, 3
loot_item = blueprint_gun_codename_repair_special01, 3
loot_item = blueprint_gun_codename_repair_special01, 3
loot_item = blueprint_gun_codename_repair_special01, 3
```
No?

Reply:
Sorry, the review summary showed me the 4 lines above, instead of below. Scratch that